public/app/features/trails/DataTrailCard.tsx

```diff
@@ -50,65 +59,168 @@ export function DataTrailCard(props: Props) {
   const { dsValue, filters, metric, createdAt } = values;
 
   return (
-    <Card onClick={onSelect} className={styles.card}>
-      <Card.Heading>{getMetricName(metric)}</Card.Heading>
-      <div className={styles.description}>
-        <Stack gap={1.5} wrap="wrap">
+    <div>
+      <Card onClick={onSelect} className={styles.card}>
+        <Card.Heading>
+          <div className={styles.metricLabel}>Metric:</div>
+          <div className={styles.metricValue}>{getMetricName(metric)}</div>
+        </Card.Heading>
+        <Card.Meta separator={'|'} className={styles.meta}>
           {filters.map((f) => (
-            <Tag key={f.key} name={`${f.key}: ${f.value}`} colorIndex={12} />
+            <span key={f.key}>
+              <div className={styles.secondaryFont}>{f.key}: </div>
+              <div className={styles.primaryFont}>{truncateValue(f.key, f.value, 44)}</div>
+            </span>
           ))}
-        </Stack>
-      </div>
-      <Card.Actions className={styles.actions}>
-        <Stack gap={1} justifyContent={'space-between'} grow={1}>
-          <div className={styles.secondary}>
-            <b>Datasource:</b> {getDataSourceName(dsValue)}
-          </div>
-          {createdAt && (
-            <i className={styles.secondary}>
-              <b>Created:</b> {dateTimeFormat(createdAt, { format: 'LL' })}
-            </i>
+        </Card.Meta>
+        <div className={styles.datasource}>
+          <div className={styles.secondaryFont}>Datasource: </div>
```

Comment:
Not sure if this matters, but the designs have "Data source:" instead of "Datasource:"
```suggestion
          <div className={styles.secondaryFont}>Data source: </div>
```

Reply:
ah great catch!